identity/handler.go

```diff
@@ -395,6 +395,11 @@ type AdminUpdateIdentityBody struct {
 	// required: true
 	Traits json.RawMessage `json:"traits"`
 
+	// Credentials represents all credentials that can be used for authenticating this identity.
+	//
+	// Use this structure to import credentials for a user.
```

Comment:
Can you please clarify that this will **completely** override the identity's credentials? If used incorrectly, this can cause a user to loose access to their account!

Reply:
Done